src/main/java/Duke.java

```diff
@@ -1,10 +1,223 @@
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println("deadline description must contain a date!");
+                    }
+
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("deadline description cannot be empty!");
+                }
+
+
+            } else if (split[0].equals("event")) {
+                try {
+                    // Check if description is empty
+                    String check = split[1];
+                    try {
+                        // Take the substring of user input after event
+                        String item = userInput.substring(6);
+                        // Divide the substring into task name and date
+                        String[] divide = item.split("/");
+                        String name = divide[0];
+                        String time = divide[1];
+                        Event e = new Event(name, LocalDate.parse(time.substring(3)));
+                        toDoList.add(e);
+                        System.out.printf("task added:\n%s", e);
+                        System.out.printf("you now have %d tasks\n", toDoList.size());
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println("event description must contain a date!");
+                    }
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("event description cannot be empty!");
+                }
+
+            } else if (split[0].equals("delete")) {
+                try {
+                    // Checks if description is empty
+                    int toDelete = Integer.parseInt(split[1]);
+                    try {
+                        Task t = toDoList.get(toDelete - 1);
+                        toDoList.remove(toDelete - 1);
+                        System.out.printf("task removed:\n%s", t.toString());
+                        System.out.printf("you now have %d tasks\n", toDoList.size());
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println(("the index you have entered does not exist!"));
+                    }
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("delete description cannot be empty!");
+                } catch (NumberFormatException e) {
+                    System.out.println("delete description must be an integer!");
+                }
+
+
+            } else {
+                // If user input does not match any commands
+                System.out.println("invalid command! try 'help' for list of commands");
+            }
+            userInput = sc.nextLine();
+            split = userInput.split(" ");
+        }
+        System.out.println("goodbye!");
+        try {
+            FileOutputStream writer = new FileOutputStream("../../../data/data.txt");
```

Comment:
Try to avoid "magic values" such as this `"../../../data/data.txt"` filepath here.

Perhaps declare it as a constant instead?